tests/test_packageAlerts.py

```diff
@@ -153,6 +204,49 @@ def setUp(self):
         self.cutoutWcs.wcs.cd = self.exposure.getWcs().getCdMatrix()
         self.cutoutWcs.wcs.ctype = ["RA---TAN", "DEC--TAN"]
 
+    def _deserialize_alert(self, alert_bytes):
+        """Deserialize an alert message from Kafka.
+
+        Parameters
+        ----------
+        alert_bytes : `bytes`
+            Binary-encoding serialized Avro alert, including Confluent Wire
+            Format prefix.
+
+        Returns
+        -------
+        alert : `dict`
+            An alert payload.
+        """
+        schema = alertPack.Schema.from_uri(
+            str(alertPack.get_uri_to_latest_schema()))
+
+        header_bytes = alert_bytes[:5]
+        version = self._deserialize_confluent_wire_header(header_bytes)
+        assert version == 0
```

Comment:
`self.assertEqual` instead?

Reply:
Missed changing that one, thanks for pointing it out. Swapped to `self.assertEqual`

Reply:
I think the purpose of that bare `assert` was to check that the version coming out of the header is always 0. I don't know if that's actually the behavior we want here, but I don't think it's a unittest-related thing, but rather a "is the code sensible" test, hence a bare assert. I might be wrong, though?